docs/user/security/securing-communications/index.asciidoc

```diff
@@ -26,65 +26,65 @@ NOTE: You do not need to enable the {es} {security-features} for this type of en
 When you obtain a server certificate, you must set its subject alternative name (SAN) correctly to ensure that modern web browsers with
 hostname verification will trust it. You can set one or more SANs to the {kib} server's fully-qualified domain name (FQDN), hostname, or IP
 address. When choosing the SAN, you should pick whichever attribute you will be using to connect to {kib} in your browser, which is likely
-the FQDN.
+the FQDN in a production environment.
 
-
-You may choose to generate a certificate signing request (CSR) and private key using the {ref}/certutil.html[`elasticsearch-certutil`] tool.
+You may choose to generate a signed certificate and private key using the [`elasticsearch-certutil`] tool.
```

Comment:
Did you intentionally remove the link to the docs for `elasticsearch-certutil`?

Reply:
Nope, I must have done a cut when I meant to copy, I'll add that back in, good catch thanks!